build.gradle.kts

```diff
@@ -1,94 +1,16 @@
-import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-import org.owasp.dependencycheck.reporting.ReportGenerator.Format.ALL
-import org.springframework.boot.gradle.plugin.SpringBootPlugin
-import java.net.InetAddress
-import java.time.Instant
-import java.time.LocalDate
-import java.time.format.DateTimeFormatter.ISO_DATE
-
 plugins {
-  kotlin("jvm") version "1.3.71"
+  id("uk.gov.justice.digital.hmpps.gradle.DpsSpringBoot") version "0.0.1-SNAPSHOT"
   kotlin("plugin.spring") version "1.3.71"
   kotlin("plugin.jpa") version "1.3.71"
-  id("org.springframework.boot") version "2.2.6.RELEASE"
-  id("io.spring.dependency-management") version "1.0.9.RELEASE"
-  id("org.owasp.dependencycheck") version "5.3.2.1"
-  id("com.github.ben-manes.versions") version "0.28.0"
-  id("com.gorylenko.gradle-git-properties") version "2.2.2"
-  id("se.patrikerdes.use-latest-versions") version "0.2.13"
-}
-
-repositories {
-  mavenLocal()
-  mavenCentral()
-}
-
-tasks.withType<KotlinCompile> {
-  kotlinOptions {
-    jvmTarget = "11"
-  }
-}
-
-java {
-  sourceCompatibility = JavaVersion.VERSION_11
-  targetCompatibility = JavaVersion.VERSION_11
-}
-
-dependencyCheck {
-  failBuildOnCVSS = 5f
-  suppressionFiles = listOf()
```

Comment:
every time I run the build I get a `dps-gradle-spring-boot-suppressions.xml` file, not sure that is correct.  This project doesn't have any spring security dependencies so doesn't need those suppressions?

Reply:
Yes the copying of the file is a bit clunky.  Needs a bit of thought.

The spring security dependencies were just the only recent example and I needed something to test it with.  It would be better if the plugin checked to see if spring security is added to the project and then generate the suppression file if so.  (And maybe similar for other dependencies - so the suppression file generation is reactive).